apps/kitchensink-react/e2e/document-list.spec.ts

```diff
@@ -11,30 +16,35 @@ test.describe('Document List', () => {
         name: `Test Author ${i}`,
         biography: `This is a test biography for author ${i}`,
       })),
+      {asDraft: false},
     )
 
     // Query for the most recent document by _updatedAt
     // (it should be at the top of the list and thus visible in the viewport for the test)
     const mostRecentDoc = await client.fetch(
-      `*[_type == "author"] | order(_updatedAt desc)[0]{_id}`,
+      `*[_id in $documentIds] | order(_updatedAt desc)[0]{_id}`,
```

Comment:
also more precision to avoid flake

Reply:
I'm curious what the flakiness was here?

Reply:
In local setups (not CI), the runners for all the jobs (webkit, chrome, dashboard-chrome, etc) run in parallel against the same dataset. one job might finish up and the document that it created might get deleted, leading to errors. so this is really just best practice, we should only work with the documents _our_ job has created.